src/autofix/autofix.ts

```diff
@@ -246,26 +271,41 @@ export default async function ({
 			const jsErrors = getJsErrors(newContent, resourcePath);
 			if (jsErrors.length) {
 				const contentWithMarkers = newContent.split("\n");
-				const message = `Syntax error after applying autofix for '${resourcePath}':\n - ` +
-					jsErrors
-						.sort((a, b) => {
-							if (a.start === undefined || b.start === undefined) {
-								return 0;
-							}
-							return a.start - b.start;
-						}).map((d) => {
-							if (d.line !== undefined && d.character !== undefined) {
-								// Insert line below the finding and mark the character
-								const line = d.line + 1;
-								contentWithMarkers.splice(line, 0, " ".repeat(d.character) + "^");
-							}
-							let res = d.messageText;
-							if (d.codeSnippet) {
-								res += ` (\`${d.codeSnippet}\`)`;
-							}
-							return res;
-						}).join("\n - ");
+				let lineOffset = 0;
+				const message = `Syntax error after applying autofix for '${resourcePath}'. ` +
+					`This is likely a UI5 linter internal issue. Please check the verbose log. ` +
+					`Please report this using the bug report template: ` +
+					`https://github.com/UI5/linter/issues/new?template=bug-report.md`;
+				const errors = jsErrors
+					.sort((a, b) => {
+						if (a.start === undefined || b.start === undefined) {
+							return 0;
+						}
+						return a.start - b.start;
+					}).map((d) => {
+						let res = d.messageText;
+						if (d.line !== undefined && d.character !== undefined) {
+							// Prepend line and character
+							res = `${d.line + 1}:${d.character + 1} ${res}`;
+
+							// Fill contentWithMarkers array for debug logging
+							// Insert line below the finding and mark the character
+							const lineContent = contentWithMarkers[d.line + lineOffset];
+							// Count number of tabs until the character
+							const tabCount = lineContent.slice(0, d.character).split("\t").length - 1;
+							const leadingTabs = "\t".repeat(tabCount);
```

Comment:
Could it be spaces or even worse- mixed style- tabs & spaces?
In such case of bad document formatting, the carret would be in the wrong position